deploy/lib/xquery/setup.xqy

```diff
@@ -4544,6 +4559,13 @@ declare function setup:get-privilege-by-name($name as xs:string) as element(sec:
     </options>)
 };
 
+declare function setup:get-users-by-name($names as xs:string*) as element(sec:users)? {
+  let $ids :=
+    for $name in $names
+      return xdmp:user($name)
```

Comment:
I am afraid my advise has had a negative side effect. You former approach would probably have silently ignored users and roles that didn't exist, whereas this will throw a large stacktrace. I lean towards ignoring it silently, you will notice soon enough it didn't capture such users/roles once you look inside the captured ml-config..